src/sentry/static/sentry/app/views/organizationCreate.jsx

```diff
@@ -26,7 +26,7 @@ export default class OrganizationCreate extends AsyncView {
     let privacyUrl = ConfigStore.get('privacyUrl');
 
     return (
-      <NarrowLayout>
+      <NarrowLayout showLogout={true}>
```

Comment:
I think you can omit the prop value and rely on attribute presence = true

```suggestion
      <NarrowLayout showLogout>
```